<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

<title>Sergio Mares</title>
<meta name="description" content="">

<!-- Open Graph -->


<!-- Bootstrap & MDB -->
<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

<!-- Fonts & Icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"  integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

<!-- Code Syntax Highlighting -->
<!-- <link rel="stylesheet" href="https://combinatronics.io/jwarby/pygments-css/master/github.css" /> -->

<!-- Styles -->
<link rel="shortcut icon" href="/assets/img/favicon.ico">
<link rel="stylesheet" href="/assets/css/main.css">

<link rel="canonical" href="/">

    
<!-- MathJax -->
<script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.1.2/es5/tex-mml-chtml.js"></script>
<script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>


  </head>

  <body class="fixed-top-nav ">

    <!-- Header -->

    <header>

    <!-- Nav Bar -->
    <nav id="navbar" class="navbar navbar-light bg-white navbar-expand-sm fixed-top">
    <div class="container">
      
      <div class="row ml-1 ml-sm-0">
        <span class="contact-icon text-center" nav="true">
          <!-- <a href="mailto:%79%69%72%75.%63%68%65%6E@%63%6F%6C%75%6D%62%69%61.%65%64%75"><i class="fas fa-envelope"></i></a> -->
          <a href="https://scholar.google.com/citations?user=RtKyA3wAAAAJ&hl=en" target="_blank" title="Google Scholar"><i class="ai ai-google-scholar"></i></a>
          <a href="https://github.com/sermare" target="_blank" title="GitHub"><i class="fab fa-github"></i></a>
          <a href="https://www.linkedin.com/in/sergio-mares/" target="_blank" title="LinkedIn"><i class="fab fa-linkedin"></i></a>
          <a href="https://x.com/sergiomaresd" target="_blank" title="Twitter"><i class="fab fa-twitter"></i></a>
        </span>

      </div>

      <!-- Navbar Toogle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <!-- About -->
          <li class="nav-item active">
            <a class="nav-link" href="/">
              home
              <span class="sr-only">(current)</span>  
            </a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="/assets/SM-2022.pdf">
              CV
            </a>
          </li>
      
          <!--<li class="nav-item ">
            <a class="nav-link" href="/publications/">
              Publications
            </a>
          </li>
      
          <li class="nav-item ">
            <a class="nav-link" href="/teaching/">
              Teaching
            </a>
          </li>-->
        </ul>
      </div>

  </nav>

</header>


    <!-- Content -->

    <div class="container mt-5">
      <div class="post">

  <header class="post-header">
    <h1 class="post-title">
     <span class="font-weight-bold">Sergio</span>   E. Mares
    </h1>
  </header>

  <article>
    
    <div class="profile float-right">
      
        <img class="img-fluid z-depth-1 rounded" src="/assets/img/avatar.png">
      
        <div class="address">
          <p><small>sergio [dot] mares [at] berkeley [dot] edu</small> </p>

        </div>
      
    </div>
    

    <div class="clearfix">
      <p>Hello! I am a fourth year Computational Biology Ph.D Candidate at UC Berkeley, advised by Professor <a href="https://www2.eecs.berkeley.edu/Faculty/Homepages/nilah.html"> Nilah Ioannidis</a> at the <a href="https://ccb.berkeley.edu">Center for Computational Biology</a> and Professor <a href="https://costellolab.ucsf.edu">Joseph Costello</a> at the UCSF Neurosurgery Department.</p>

      <p>I have a B.S in Biochemistry and a B.S in Microbiology/Cell and Molecular Biology with minors in Chemistry, Plant Biology, and Spanish from <a href="https://news.okstate.edu/articles/arts-sciences/2021/sergio_mares.html">Oklahoma State University</a>, where I was advised by Professor <a href="https://scholar.google.com/citations?user=Pae30scAAAAJ&hl=en">Marianna Patrauchan</a> at the Microbiology and Molecular Genetics Department, where I worked on the conservation of calcium signalling proteins in pathogenic microbes.</p>
      
      <p>Previously, I have also worked with Professor <a href="https://scholar.google.com/citations?user=jlRDnO8AAAAJ&hl=en">Humberto Ortiz-Zuazaga</a> at the University of Puerto Rico - Rio Piedras, improving assembly algorithms for detecting sparse microbial populations in metagenomic datasets.</p>
      
      <p>I am broadly interested in the intersection between cancer genetics, deep learning models, and the improvement of current therapeutic strategies in cancer. Some of my current work focuses on benchmarking and generating new models for identifying novel molecular immunotherapuetic targets against brain tumors.</p>
<!--
Write your biography here. Tell the world about yourself. Link to your favorite [subreddit](http://reddit.com){:target="\_blank"}. You can put a picture in, too. The code is already in, just name your picture `prof_pic.jpg` and put it in the `img/` folder.

Put your address / P.O. box / other info right below your picture. You can also disable any these elements by editing `profile` property of the YAML header of your `_pages/about.md`. Edit `_bibliography/papers.bib` and Jekyll will render your [publications page](/al-folio/publications/) automatically.

Link to your social media connections, too. This theme is set up to use [Font Awesome icons](http://fortawesome.github.io/Font-Awesome/){:target="\_blank"} and [Academicons](https://jpswalsh.github.io/academicons/){:target="\_blank"}, like the ones below. Add your Facebook, Twitter, LinkedIn, Google Scholar, or just disable all of them. -->


    </div>    
      <div class="publications">

  <h2>publications</h2>

<ol class="bibliography">

    <li><div class="row">
      <div class="col-sm-3 float-left">
        <center>
        <img class="img-fluid z-depth-1 rounded" src="/assets/img/pic010.png">
        <!-- <abbr class="badge">CoLLAs 2023</abbr> -->
        </center>
      </div>
    
      <div id="fan2023dpok" class="col-sm-7">
        
        <div class="title"><a href="https://doi.org/10.1099/mic.0.001004">Generation of structure-guided pMHC-I libraries using Diffusion Models</a></div>
        <div class="author">
        <b>Sergio E. Mares</b>, Ariel Espinoza, Nilah M. Ioannidis
      
      </div>
    
          <div class="periodical">
          
            <em>Internatinal Conference on Machine Learning Gen AI and Biology Workshop 2025</em>
          
          </div>
        
    
        <div class="links">
          <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
          <a href="https://openreview.net/pdf?id=NUc1ZMvRim" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
          <a href="/assets/img/paper_ICML.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
        </div>
    
        <!-- Hidden abstract block -->
        
        <div class="abstract hidden">
          <p>Personalized vaccines and T-cell immunotherapies depend critically on identifying peptide-MHC class I (pMHC-I) interactions capable of eliciting potent immune responses. However, current benchmarks and models inherit biases present in mass-spectrometry and binding-assay datasets, limiting discovery of novel peptide ligands. To address this issue, we introduce a structure-guided benchmark of pMHC-I peptides designed using diffusion models conditioned on crystal structure interaction distances. Spanning twenty high-priority HLA alleles, this benchmark is independent of previously characterized peptides yet reproduces canonical anchor residue preferences, indicating structural generalization without experimental dataset bias. Using this resource, we demonstrate that state-of-the-art sequence-based predictors perform poorly at recognizing the binding potential of these structurally stable designs, indicating allele-specific limitations invisible in conventional evaluations. Our geometry-aware design pipeline yields peptides with high predicted structural integrity and higher residue diversity than existing datasets, representing a key resource for unbiased model training and evaluation.</p>
        </div>
        
      </div>
    </div>
    </li>


        
  <ol class="bibliography">

    <li><div class="row">
      <div class="col-sm-3 float-left">
        <center>
        <img class="img-fluid z-depth-1 rounded" src="/assets/img/pic010.png">
        <!-- <abbr class="badge">CoLLAs 2023</abbr> -->
        </center>
      </div>
    
      <div id="fan2023dpok" class="col-sm-7">
        
        <div class="title"><a href="https://doi.org/10.1099/mic.0.001004">carP, encoding a Ca2+-regulated putative phytase, is evolutionarily conserved in Pseudomonas aeruginosa and has potential as a biomarker</a></div>
        <div class="author">
        <b>Sergio E. Mares</b>, M. King, A. Kubo, A. Khavov, E. Lutter, N. Youssef, M. Patrauchan
      
      </div>
    
          <div class="periodical">
          
            <em>Journal of Microbiology 2020</em>
          
          </div>
        
    
        <div class="links">
          <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
          <a href="https://doi.org/10.1099/mic.0.001004" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
          <a href="/assets/img/paper.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
        </div>
    
        <!-- Hidden abstract block -->
        
        <div class="abstract hidden">
          <p>Here, we studied the conservation of carP sequence and its occurrence in diverse phylogenetic groups of bacteria. In silico analysis revealed that carP and its two paralogues PA2017 and PA0319 are primarily present in P. aeruginosa and belong to the core genome of the species. We identified 155 single nucleotide alterations within carP, 42 of which lead to missense mutations with only three that affected the predicted 3D structure of the protein. PCR analyses with carP-specific primers detected P. aeruginosa specifically in 70 clinical and environmental samples. Sequence comparison demonstrated that carP is overall highly conserved in P. aeruginosa isolated from diverse environments. Such evolutionary preservation of carP illustrates its importance for P. aeruginosa adaptations to diverse environments and demonstrates its potential as a biomarker.</p>
        </div>
        
      </div>
    </div>
    </li>

    <li><div class="row">
      <div class="col-sm-3 float-left">
        <center>
        <img class="img-fluid z-depth-1 rounded" src="/assets/img/output.gif">
        <!-- <abbr class="badge">CoLLAs 2023</abbr> -->
        </center>
      </div>
    
      <div id="du2023vlm" class="col-sm-7">
        
          <div class="title"><a href="https://doi.org/10.1038/s41598-022-12584-9">EF-hand protein, EfhP, specifically binds Ca2+ and mediates Ca2+ regulation of virulence in a human pathogen Pseudomonas aeruginosa</a></div>
          <div class="author">
            Biraj B. Kayastha, A. Kubo, J. Burch-Konda, R. L. Dohmen, J. L. McCoy, R. R. Rogers,<b> Sergio E. Mares</b>, J. Bevere, A. Huckaby, W. Witt, S. Peng, B. Chaudhary, S. Mohanty, M. Barbier, G. Cook, J. Deng, M. Patrauchan
          </div>
    
          <div class="periodical">
          
            <em> Nature Scientific Reports 2022</em> <br>
          
          </div>
        
    
        <div class="links">
        
          <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> 
          <a href="https://doi.org/10.1038/s41598-022-12584-9" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
          <a href="/assets/img/s41598-022-12584-9.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
        </div>
    
        <!-- Hidden abstract block -->
        
        <div class="abstract hidden">
          <p>Calcium is well known as a second messenger in eukaryotes, where Ca2+ signaling controls life-sustaining cellular processes. Although bacteria produce the components required for Ca2+ signaling, little is known about the mechanisms of bacterial Ca2+ signaling. Previously, we have identified a putative Ca2+-binding protein EfhP (PA4107) and CalC as proteins involved in the calcium network.</p>
        </div>
        
      </div>
    </div>
    </li>
    
    <li><div class="row">
      <div class="col-sm-3 float-left">

        <center>
        <img class="img-fluid z-depth-1 rounded" src="/assets/img/output2.gif">
        <!-- <abbr class="badge">ICML 2023</abbr> -->
        </center>
      </div>
    
      <div id="du2023ellm" class="col-sm-7">
          <div class="title"><a href="https://doi.org/10.1091/mbc.E20-11-0705">Baculovirus actin-rearrangement-inducing factor ARIF-1 induces the formation of dynamic clusters of invadosome structures</a></div>
          <div class="author">
            Domokos I. Lauko, Taro Ohkawa, <b>Sergio E. Mares</b>, Matthew D. Welch
          </div>
    
          <div class="periodical">
          
            <em>Molecular Biology of the Cell 2021</em>
          
          </div>
        
    
        <div class="links">
        
          <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
          <a href="https://doi.org/10.1091/mbc.E20-11-0705" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
          <a href="/assets/img/baco.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>

        </div>
    
        <!-- Hidden abstract block -->
        
        <div class="abstract hidden">
          <p>Studies have shown that the AcMNPV protein actin rearrangement inducing factor-1 (Arif-1) is necessary and sufficient to induce the formation of cortical concentrations of polymerized actin, termed ventral aggregates (VAs), in cultured insect cells. Bombyx mori nucleopolyhedrosis virus (BmNPV), plays a role in enhancing systemic infection in B. mori larvae. However, it is still unknown if BmNPV Arif-1 is sufficient to cause the formation of VAs, or what role membrane anchoring regions of Arif-1 play in this process.</p>
        </div>
        
      </div>
    </div>
    </li>

  <li><div class="row">
      <div class="col-sm-3 float-left">

        <center>
        <img class="img-fluid z-depth-1 rounded" src="/assets/img/pic050.png">
        <!-- <abbr class="badge">ICRA 2023</abbr> -->
      </center>
      </div>
    
      <div id="khansari2023tcl" class="col-sm-7">
        
          <div class="title"><a href="https://journals.asm.org/doi/10.1128/aem.01706-21">Genomes of novel Myxococcota reveal severely curtailed machineries for predation and cellular differentiation</a></div>
          <div class="author">
            Chelsea L. Murphy, R. Yang, T. Decker, C. Cavalliere, V. Andreev, N. Bircher, J.Cornell, R. Dohmen, C. J. Pratt, A. Grinnell, J. Higgs, C. Jett, E. Gillett, R. Khadka,<b> Sergio E. Mares</b>, C. Meili, J. Liu, H. Mukhtar, Mostafa S. Elshahed, Noha H. Youssef
          </div>
    
          <div class="periodical">
          
            <em> Environmental Microbiology 2021</em>
          
          </div>
        
    
        <div class="links">
        
          <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
          <a href="https://journals.asm.org/doi/10.1128/aem.01706-21" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
          <a href="/assets/img/paper2.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
        </div>
    
        <!-- Hidden abstract block -->
        
        <div class="abstract hidden">
          <p>Detailed analysis of 13 distinct pathways/processes crucial to predation and cellular differentiation revealed severely curtailed machineries, with the notable absence of homologs for key transcription factors, outer membrane exchange receptor (TraA), and the majority of sporulation-specific and A-motility-specific genes. We propose that predation and cellular differentiation represent a niche adaptation strategy that evolved circa 500 million years ago (Mya) in response to the rise of soil as a distinct habitat on Earth.</p>
        </div>
        
      </div>
    </div>
    </li>
    
</div>

    </div>

    <!-- Footer -->

    
<footer class="fixed-bottom">
  <div class="container mt-0">
    &copy; Copyright 2023 Sergio Mares.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>.

    
    Last updated: June 2024.
    
  </div>
</footer>



  </body>

  <!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>

  <!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>

  
<!-- Mansory & imagesLoaded -->
<script defer src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
<script defer src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
<script defer src="/assets/js/mansory.js" type="text/javascript"></script>


  



<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>


</html>
